app/api/api/entities/donation.rb

```diff
@@ -22,7 +22,9 @@ class Donation < LinkedObjectBase
           ]
         }
         expose :recurring?, as: :recurring, documentation: { type: "boolean" }
-
+        expose :avatar_url do |donation, _options|
+          donation.local_hcb_code&.fallback_avatar
```

Comment:
```suggestion
          gravatar_url(donation.email, donation.name, donation.email.to_i, 48) unless donation.anonymous?
```

We can probably skip going through `local_hcb_code`